src/qml/qgismobileapp.qml

```diff
@@ -630,8 +631,8 @@ ApplicationWindow {
     /** A rubberband for the different geometry editors **/
     Rubberband {
       id: geometryEditorsRubberband
-      lineWidth: 2.5
-      color: '#80000000'
+      lineWidth: 4
+      color: '#96000000'
```

Comment:
Since these are defaults, would be easy if we use them instead of explicitly setting `lineWidth` in several places?